packages/batcher/utils/src/config.ts

```diff
@@ -127,7 +127,31 @@ export class ENV {
     return process.env.SELF_SIGNING_API_KEY || '';
   }
 
-  private static isTrue(value: string | undefined): boolean {
+  // MQTT BROKER
+  static get MQTT_BROKER(): boolean {
+    return ENV.isTrue(process.env.MQTT_BROKER, true);
+  }
+  static get MQTT_ENGINE_BROKER_PORT(): number {
+    return parseInt(process.env.MQTT_BROKER_PORT || '8883', 10);
+  }
+  static get MQTT_BATCHER_BROKER_PORT(): number {
+    return parseInt(process.env.MQTT_BROKER_PORT || '8884', 10);
+  }
+  // MQTT CLIENT
+  static get MQTT_ENGINE_BROKER_URL(): string {
+    return process.env.MQTT_ENGINE_BROKER_URL || 'ws://127.0.0.1:8883';
```

Comment:
As an example of the above comment concretely, we shouldn't have `8883` here if the port is meant to be defined by the above ENV variable